### PR TITLE
iOS Only - Re-arrange scroll position of text inputs on landscape mode

### PR DESCRIPTION
**Pull request by pinarol**
This PR re-arranges the scrolling offset of text inputs for landscape mode.

We are showing the inner toolbar in portrait mode but in landscape mode we don't have room for it so we won't show that.

Before:

ezgif com-video-to-gif 1

After:
ezgif com-video-to-gif 2

Fixes part of https://github.com/wordpress-mobile/gutenberg-mobile/issues/197

**To Test**

Test 1 - Landscape mode

- Open a blog post on WPiOS app
- Switch to landscape mode
- Add one of the text input blocks with the return button on keyboard
- Verify that focused field is viewable
- Add one of the text input blocks with (+) button
- Verify that focused field is viewable
- Close the keyboard, focus again on one of the text input blocks
- Verify that focused field is viewable

Test 2 - Portait mode

- Open a blog post on WPiOS app
- Switch to portrait mode
- Add one of the text input blocks with the return button on keyboard
- Verify that focused field is viewable with the inner toolbar
- Add one of the text input blocks with (+) button
- Verify that focused field is viewable with the inner toolbar
- Close the keyboard, focus again on one of the text input blocks
- Verify that focused field is viewable with the inner toolbar

### CODE REVIEW

**Comment by pinarol** [comment]
@marecar3 I've updated the import with this PR that you mentioned in https://github.com/wordpress-mobile/gutenberg-mobile/pull/387#discussion_r242525865 I've already tested it with WP-Android but feel free to test it yourself, it'd be nice to have a double check.

File: src/components/keyboard-aware-flat-list.android.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 * @format
 * @flow
 */
-import { FlatList, KeyboardAvoidingView } from 'react-native';
+import { FlatList } from 'react-native';
+import KeyboardAvoidingView from '../components/keyboard-avoiding-view';
```
Comment (pinarol) [inline src/components/keyboard-aware-flat-list.android.js]: to address https://github.com/wordpress-mobile/gutenberg-mobile/pull/387#discussion_r242525865

**Comment by iamthomasbishop** [comment]
Thanks for tackling this @pinarol! One minor thing that I noticed: can we add the same width/color border on the left/right sides, regardless of orientation?

**Comment by pinarol** [comment]
@iamthomasbishop I've opened a new issue to address that https://github.com/wordpress-mobile/gutenberg-mobile/issues/402 I am not sure that's a simple to fix within this PR but I'll try to fix that as my next task.

**Pull request merged**